a10_neutron_lbaas/v2/handler_pool.py

```diff
@@ -84,3 +83,35 @@ def delete(self, context, pool):
 
             handler_persist.PersistHandler(
                 c, context, pool, self._meta_name(pool)).delete()
+
+    def _update_session_persistence(self, old_pool, pool, c, context):
+        # import pdb; pdb.set_trace()
+        # didn't exist, does exist, create
+        if not old_pool or (not old_pool.session_persistence and pool.session_persistence):
+            p = handler_persist.PersistHandler(c, context, pool, old_pool)
+            p.create()
+            return
+
+        # existed, change, delete and recreate
+        if (old_pool.session_persistence and pool.session_persistence and
+                old_pool.session_persistence.type != pool.session_persistence.type):
+            p = handler_persist.PersistHandler(c, context, old_pool)
+            p.delete()
+            p = handler_persist.PersistHandler(c, context, pool)
+            p.create()
+            return
+
+        # didn't exist, does exist now, create
+        if old_pool.session_persistence and not pool.session_persistence:
+            p = handler_persist.PersistHandler(c, context, pool)
+            p.create()
+            return
+
+        # didn't exist, doesn't exist
```

Comment:
from 110 down, you can just put in the two comments and not the if conditionals, as they do nothing.